docs/using/go.rst

```diff
@@ -30,7 +30,7 @@ world tutorial-app, let's call it "helloworld":
 
 ::
 
-    $ tsuru app create helloworld go
+    $ tsuru app create helloworld go --team
```

Comment:
this is not accurate, if the user runs this exact command tsuru will ask the user to specify a valid team